README.md

```diff
@@ -1,9 +1,17 @@
 # JSOSolvers.jl
 
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3991143.svg)](https://doi.org/10.5281/zenodo.3991143)
+[![GitHub release](https://img.shields.io/github/release/JuliaSmoothOptimizers/JSOSolvers.jl.svg)](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl/releases/latest)
+[![](https://img.shields.io/badge/docs-stable-3f51b5.svg)](https://JuliaSmoothOptimizers.github.io/JSOSolvers.jl/stable)
+[![](https://img.shields.io/badge/docs-latest-3f51b5.svg)](https://JuliaSmoothOptimizers.github.io/JSOSolvers.jl/latest)
+[![codecov](https://codecov.io/gh/JuliaSmoothOptimizers/JSOSolvers.jl/branch/master/graph/badge.svg?token=eyiGsilbZx)](https://codecov.io/gh/JuliaSmoothOptimizers/JSOSolvers.jl)
+
 ![CI](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl/workflows/CI/badge.svg?branch=master)
 [![Cirrus CI - Base Branch Build Status](https://img.shields.io/cirrus/github/JuliaSmoothOptimizers/JSOSolvers.jl?logo=Cirrus%20CI)](https://cirrus-ci.com/github/JuliaSmoothOptimizers/JSOSolvers.jl)
-[![codecov](https://codecov.io/gh/JuliaSmoothOptimizers/JSOSolvers.jl/branch/master/graph/badge.svg?token=eyiGsilbZx)](https://codecov.io/gh/JuliaSmoothOptimizers/JSOSolvers.jl)
-[![](https://img.shields.io/badge/docs-latest-3f51b5.svg)](https://JuliaSmoothOptimizers.github.io/JSOSolvers.jl/latest)
 
 This package provides optimization solvers curated by the JuliaSmoothOptimizers
 organization.
+
+## How to cite
+
+If you use JSOSolvers.jl in your work, please cite using the format given in [CITATION.bib](CITATION.bib).
```

Comment:
Does this link work? In other repos, we use a path to the master/main branch.

Reply:
just checked, it does yes